src/components/FruitCounter.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import './FruitCounter.css';
+
+
+export function fruitCounter ({emoji, title, setCounter, counter}){
+
+    function decreaseCounter() {
+        setCounter(counter - 1)
```

Comment:
Nu zal telkens wanneer `counter` veranderd een onnodige rerender van deze functie plaatsvinden. Wat je zou kunnen doen is de huidige waarde gebruiken die je vanuit de setCounter functie al krijgt:
```suggestion
        setCounter(currentValue => currentValue - 1)
```